File: app/src/main/java/com/kksionek/gdzietentramwaj/GridViewItem.java
package com.kksionek.gdzietentramwaj;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.RelativeLayout;

public class GridViewItem extends RelativeLayout {

    public GridViewItem(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);
    }
}

File: app/src/main/java/com/kksionek/gdzietentramwaj/favorite/view/GridViewItem.kt
package com.kksionek.gdzietentramwaj.favorite.view

import android.content.Context
import android.util.AttributeSet
import android.widget.RelativeLayout

class GridViewItem @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defAttrStyle: Int = 0
) : RelativeLayout(context, attrs, defAttrStyle) {

    public override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }
}